### DISCUSSION TITLE
[feat] Adjust fallback font sizes

### DISCUSSION THREAD

**Discussion by KorigamiK** [ghostty-org/ghostty#3296] [Feature Requests, Ideas] [answered]
It can be noted how [foot](https://codeberg.org/dnkl/foot) does font configuration:

```sh
font=Zed Mono:size=15, Symbols Nerd Font:size=11, Noto Color Emoji:size=12
```

Which allows you to give sizes for individual fonts.

I want to achieve the size that kitty renders the symbols as.

| Ghostty    | Kitty |
| -------- | ------- |
| ![Image](https://github.com/user-attachments/assets/dc9309d8-ccaf-485b-9f9d-5a28e52725ff)  | ![Image](https://github.com/user-attachments/assets/b4ff96c3-3b0a-4e7e-b6f5-9f2bbd5e78b6) |

**Comment by mitchellh** [comment]
I’m in agreement I’d like to support this. We need to discuss what this looks like in our config so that issue can be actionable by a contributor.

**Comment by swarn** [comment]
This has the additional effect currently that fallback fonts are ignored when they don't have the options specified for the main font. Given
```
font-family = SF Mono
font-style = Medium
font-style-bold = Bold
font-style-italic = Medium Italic
font-style-bold-italic = Bold Italic
font-size = 13
adjust-cell-width = -10%

font-family = Symbols Nerd Font Mono
```
Then,
```sh
ghostty +show-face --cp=0x276f
```
```
U+276F « ❯ » found in face “JetBrainsMono Nerd Font”.
```
But it will fall back correctly if `font-style-*` are not specified.

One possibility for config syntax, if you want to continue avoiding nesting, is to have the `font-*` options only apply to the most recent `font-family`.

**Comment by 00-kat** [comment, marked as answer]
Actually I only just realised, this is a duplicate of  #2973.
